lib/fuser_basic.h

```diff
@@ -57,15 +51,17 @@ struct BasicGateFuser final {
   };
 
   /**
-   * Stores ordered sets of gates, each acting on two qubits, that can be
-   * applied together. Note that gates fused with this method are not
+   * Stores sets of gates that can be applied together. Only one- and
+   * two-qubit gates are get fused. Gates fused with this method are not
```

Comment:
"are get" -> "will get"

Same elsewhere in file.

Reply:
Done.